### PR TITLE
Adding hyperapp-multiselect

### PR DESCRIPTION
**Pull request by congwenma**

### CODE REVIEW

**Comment by congwenma** [comment]
@frenzzy @jorgebucaran For some reason I can't add reviewers to this PR, What do you guys think about the Select widget I created?

**Comment by jorgebucaran** [comment]
Hi @congwenma, thank you for your submission. Can you summarize what this is and how it works? A demo would be awesome. Also, please make sure this targets V2 as I don't maintain V1 anymore.

**Comment by zaceno** [comment]
from congwenma/hyperapp-multiselect repo it looks to be for V1

**Comment by jorgebucaran** [comment]
I just realized this is a very old PR. I'd be more than happy to merge this when and if it's rewritten for V2.
